view/next-project/src/components/sponsoractivities/EditModal.tsx

```diff
@@ -15,6 +15,7 @@ import {
 import { MultiSelect } from '@components/common';
 import { BUREAUS } from '@constants/bureaus';
 import { SponsorActivity, Sponsor, SponsorStyle, User, ActivityStyle } from '@type/common';
+import { DESIGNERS, DESIGNER_VALUES } from "@constants/designers";
```

Comment:
使ってないimportは消して大丈夫（VSCodeだと薄く灰色になっていると思う）
これも@typeより@constantsディレクトリの方がimport順が上だね

```suggestion
import { DESIGNER_VALUES } from "@constants/designers";
import { SponsorActivity, Sponsor, SponsorStyle, User, ActivityStyle } from '@type/common';
```